### ISSUE TITLE
Support for Headphone Remote (cable)

### ISSUE THREAD

**Issue by Tekl** [treasurebox/MagicKeys#12]
What about adding support Earphone controls like in the prefpane from VOX?

http://coppertino.com/addon

**Comment by treasurebox** [comment]
Hi!

  This shoud work now with Apple earbuds. So it's not working, or you are using some other kind of earbuds? What is your OS X version?

**Comment by Tekl** [comment]
I tried the Bose SoundTrue and also Apples EarPods. I can use the play button to launch the selected App but I can't play/pause any player on my system (OS 10.10.5). When I disable MagicKeys I can control iTunes again. Volume Control works as expected.

**Comment by treasurebox** [comment]
Are you using it to control Gear, or something else? As the routing of media keys is somewhat complicated, sometimes (for unknown reason) it does not work on some configurations.
Generally I advise users with similar question to use Gear's out of App Store version in this case, of course if you are using MK for something else, this is out of the question.

**Comment by Tekl** [comment]
I don't use Gear. I'm just looking for a solution to make the media keys available in every app (iTunes, MplayerX, Spotify ...). But anyway thanks for your support.

**Comment by forivall** [comment]
Yeah, i'm using this with spotify, and when i have magic keys disabled, the button works, but also opens up itunes, and then sends play/pause to both itunes and spotify. when enabled, nothing happens.

the headphones are https://www.monoprice.com/product?p_id=14088

**Comment by j-bn** [comment]
Would also be very happy to see this! I route it through Chrome to control Play Music and it works as expected with (touch bar) media keys but, as above, the headphone play/pause button on my OnePlus Bullets V2 headphones starts iTunes (whilst also play/pausing as intended). The volume buttons do not start iTunes and correctly just alter the volume.
